docs/configuration_and_usage.rst

```diff
@@ -0,0 +1,106 @@
+Configuration and usage
+=======================
+
+Getting around the administrator interface
+------------------------------------------
+
+``Edx-enterprise`` is integrated with edx-platform Django admin. It can be accessed at
+``$LMS_SERVER_ADDRESS/admin/enterprise``. Current administrator interface is considered provisional until an independent
+full-featured enterprise portal is built, so not much time was spent on improving admin tools UX.
+
+.. image:: images/enterprise-admin.png
+
+So far, there are four items in the admin interface:
+
+* "Enrollment notification email templates" - manages templates used to build enrollment notifications.
+* "Enterprise Customers" - lists and manages Enterprise Customer records.
+* "Enterprise Customer Users" - lists and manages enterprise customer students.
+* "Pending Enterprise Customer Users" - lists and manages "pending" enterprise customer students.
```

Comment:
^ This is out of scope, but I wonder if we should just hide "Enterprise Customer Users" and "Pending Enterprise Customer Users" from the admin. Do they offer any value (compared to the Manage Learners view)?